### ISSUE TITLE
NanaZip not showing in Windows 11 "Show more options"

### ISSUE THREAD

**Issue by Raymai97** [M2Team/NanaZip#26]
Windows 11, version 10.0.22000.258
NanaZip 1.0 Preview 3

![image](https://user-images.githubusercontent.com/3296077/137838917-6cdfa04c-97c6-4ddc-b63f-2bc43e3f6ee5.png)

**Comment by bwat47** [comment]
I see the same issue with the context menus on windows 10 (which I guess makes sense, since that's the same as the windows 11 show more options menu)

If I right click on a folder I do see nanazip in the context menu

If I right click an individual file I do not

**Comment by MouriNaruto** [comment]
@Raymai97 @SakuraNeko @Centazi  @bwat47 

I think it may be the issue from Windows. I have met the similar situation when I installed NanaZip. I have solved it by restarting the File Explorer process in the Task Manager.

I have noticed to the known issues. https://github.com/M2Team/NanaZip#known-issues

So I closed the issue, and if you met the similar situation and not solved even after restarting the File Explorer process in the Task Manager. Feel free to open a new issue.

Kenji Mouri
